examples/response_stream/public/containers/app/pages/page_reducer_stream/index.tsx

```diff
@@ -144,6 +145,13 @@ export const PageReducerStream: FC = () => {
           onChange={(e) => setSimulateErrors(!simulateErrors)}
           compressed
         />
+        <EuiCheckbox
+          id="responseStreamCompressionCheckbox"
+          label="Toggle compression setting for response stream."
```

Comment:
I assume this turns compression on or off? In which case, this label might be clearer if it said something like 'Enable compression'. Otherwise it isn't clear to me if ticking the checkbox is turning it on or off.